components/lib/crash/src/main/java/mozilla/components/lib/crash/service/SentryService.kt

```diff
@@ -88,19 +90,25 @@ class SentryService(
                 .withLevel(Event.Level.INFO)
 
             client.sendEvent(eventBuilder)
+            client.context.clearBreadcrumbs()
 
             return eventBuilder.event.id.toString()
         }
 
         return null
     }
 
-    override fun report(throwable: Throwable): String? {
+    override fun report(throwable: Throwable, breadcrumbs: ArrayList<crashBreadCrumbs>): String? {
+        breadcrumbs.forEach {
+            client.context.recordBreadcrumb(it.toSentryBreadcrumb())
+        }
+
         val eventBuilder = EventBuilder().withMessage(createMessage(throwable))
                 .withLevel(Event.Level.INFO)
                 .withSentryInterface(ExceptionInterface(throwable))
 
         client.sendEvent(eventBuilder)
+        client.context.clearBreadcrumbs()
```

Comment:
👍 

Can you double check the breadcrumbs still get out. Now I am afraid that Sentry will read them async later and by then we already have cleared them. 😅 Other than that 👍 👍 👍

Reply:
Confirmed breadcrumb are still being sent 👍